code-challenges.js

```diff
@@ -20,26 +20,62 @@ const fruit2 = "banana"
 // Expected outcome: "cherry"
 
 // Pseudo code:
-
+// input: first string -> fruit1 = "apple", second string -> fruit2 = "banana"
+// output: The string with the most characters is "banana"
+// use a condition statement to evaluate the strings
+// use .length to get number of characters
+// use realtional operator to compare
 
 // --------------------1) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+// Code used: 
+console.log(padres1984WorldSeriesRuns.length + padres1998WorldSeriesRuns.length)
```

Comment:
The prompt says the code should combine the two arrays and then return the new length. Here, you are adding the 2 lengths of the arrays together instead of combining them. Make sure you pay attention to the details!